src/main/java/org/sagebionetworks/bridge/exporter/synapse/SynapseStatusTableHelper.java

```diff
@@ -0,0 +1,141 @@
+package org.sagebionetworks.bridge.exporter.synapse;
+
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.sagebionetworks.client.exceptions.SynapseException;
+import org.sagebionetworks.repo.model.table.ColumnModel;
+import org.sagebionetworks.repo.model.table.ColumnType;
+import org.sagebionetworks.repo.model.table.PartialRow;
+import org.sagebionetworks.repo.model.table.PartialRowSet;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import org.sagebionetworks.bridge.exporter.exceptions.BridgeExporterException;
+import org.sagebionetworks.bridge.exporter.worker.ExportTask;
+import org.sagebionetworks.bridge.exporter.worker.ExportWorkerManager;
+
+/**
+ * Highly specialized helper, which writes the status row to the Synapse status table, creating it if it doesn't
+ * already exist.
+ */
+@Component
+public class SynapseStatusTableHelper {
+    // Constants package-scoped to be available to unit tests.
+    static final String COLUMN_NAME_UPLOAD_DATE = "uploadDate";
+    static final List<ColumnModel> COLUMN_LIST;
+    static {
+        // Construct column definition list. This table is only used as a flag to signal that we're done uploading, so
+        // we only need one column with the date.
+        // NOTE: ColumnType.DATE is actually a timestamp. There is no calendar date type.
+        ColumnModel uploadDateColumn = new ColumnModel();
+        uploadDateColumn.setName(COLUMN_NAME_UPLOAD_DATE);
+        uploadDateColumn.setColumnType(ColumnType.STRING);
+        uploadDateColumn.setMaximumSize(10L);
```

Comment:
We don't ever include a time component, because that wouldn't be useful. (That and the Synapse timestamp column isn't an ISO timestamp and is difficult to query on, so we want to avoid that anyway.) The idea is to signal that Bridge-EX has finished running for a given day. Whether that's at 2:15am or 2:17am is inconsequential.

We do have plans to make Bridge-EX run more frequently (perhaps hourly), but this is such an in-depth change across multiple systems that what we do here doesn't really have an impact.

As for how/where Chris is recording his results, that's outside the scope of Bridge-EX. Currently, Bridge-EX only exports data from Bridge to Synapse. We may have need in the future to process Synapse data back into Synapse, but that's another project that's far outside the scope of this change.

As for whether the data should be written back into the Status Table: In the specific case, no, because the data Chris is generating is not keyed off of upload date. In the general case, see my other response.